logger/echo_logger.go

```diff
@@ -25,7 +25,7 @@ type EchoLogger struct {
 	*logrus.Entry
 }
 
-/// Wrapping _level_j methods
+// / Wrapping _level_j methods
```

Comment:
For this file - can we just make `///` -> `//`? the syntax of space between the final `/` is kind of weird.

Reply:
yeah, I covered it but I forgot to push it, good eye, thanks!